src/d/llvm/evaluator.d

```diff
@@ -379,7 +379,10 @@ struct JitRepacker {
 
 		CompileTimeExpression[] elements;
 
-		foreach (uint i, f; s.fields) {
+		assert(cast(uint) s.fields.length == s.fields.length);
+		foreach (size_t idx, f; s.fields) {
+			auto i = cast(uint) idx;
```

Comment:
or should I move the `assert` into the loop’s body (so that whatever the index may be will get checked)?

Reply:
Yes, why not?

Reply:
alright